src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getAPIScores, setAPIScores } from './modules/hit-api.js';
+import { createScoreListElement, appendListElement, clearForms } from './modules/scores.js';
+import Score from './modules/singleScore.js';
+
+const addToLocalStorage = (scores) => {
+  localStorage.setItem('scores', JSON.stringify(scores));
+};
+
+const addScoreList = document.getElementById('add');
+const refreshScoreList = document.getElementById('refresh');
+
+const scores = new Score();
+
+addScoreList.addEventListener('click', () => {
+  const newName = document.getElementById('user').value;
+  const newScore = document.getElementById('score').value;
+
+  setAPIScores(newName, newScore);
+
+  const listElement = createScoreListElement(newName, newScore);
+
+  appendListElement(listElement);
+  addToLocalStorage();
+  clearForms();
+});
+
+const updateLocalStorage = (scores) => {
+  appendListElement(scores);
+  localStorage.clear();
+  addToLocalStorage(scores);
+};
+
+const fromLocalStorage = () => {
+  if (localStorage.length !== 0) {
+    const scoresStored = JSON.parse(localStorage.getItem('scores'));
+    scoresStored.listElement.forEach((user) => {
+      scores.listElement.push(user);
+    });
+    updateLocalStorage(scores);
+  }
+  refreshScoreList();
+};
+
+fromLocalStorage();
```

Comment:
- [ ] Please make sure that, if you want to use localStorage (which is not a requirement), make sure to make it work. This is what I can see when I open the local storage after submitting a name and a score:
![Captura de pantalla 2022-02-04 193606](https://user-images.githubusercontent.com/59100477/152584265-92fc27a2-150b-4657-b974-d411c6015a36.jpg)